src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Commands/UpdateEnglishFractions/WhenISendTheUpdateEnglishFractionsCommand.cs

```diff
@@ -39,14 +39,14 @@ public void Arrange()
                 new DasEnglishFraction
                 {
                     Id = "1",
-                    DateCalculated = DateTime.Parse(DateTime.Now.AddDays(-20).ToShortDateString()),
+                    DateCalculated = DateTime.Parse(DateTime.Today.AddDays(-20).ToShortDateString()),
```

Comment:
Date time is being converted to a string and immediately converted back to a date time. Not clear why. It looks  like it might be trying to just remove the time, but if so would it be worth considering:

DateTime.Today.AddDays(-20).Date, though it's a date-only anyway (as its using .Today) so not sure what the intent is.

Reply:
Agreed, missed this whilst bulk updating code.